cve_bin_tool/strings.py

```diff
@@ -16,11 +16,11 @@ class Strings:
     # add tab to the printable character
     PRINTABLE.add(9)
 
-    def __init__(self, filename=""):
-        self.filename = filename
-        self.output = ""
+    def __init__(self, filename: str = "") -> None:
+        self.filename: str = filename
+        self.output: str = ""
 
-    async def aio_parse(self):
+    async def aio_parse(self) -> str:
         async with FileIO(self.filename, "rb") as f:
             tmp = []
```

Comment:
```suggestion
            tmp = list[str]()
```

Reply:
Sorry, forgot that you can only write like that in Python 3.9+, we'll need to do it like this for now:
```py
from typing import List
...
tmp: List[str] = []
```

Reply:
Was going to ask about that, thanks.